pkg/cmd/pulumi/history.go

```diff
@@ -15,24 +15,15 @@
 package main
 
 import (
-	"encoding/json"
-	"fmt"
-	"sort"
-	"strings"
-	"time"
-
-	"github.com/dustin/go-humanize"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
-	"github.com/pulumi/pulumi/pkg/v2/backend"
 	"github.com/pulumi/pulumi/pkg/v2/backend/display"
-	"github.com/pulumi/pulumi/sdk/v2/go/common/diag/colors"
 	"github.com/pulumi/pulumi/sdk/v2/go/common/resource/config"
 	"github.com/pulumi/pulumi/sdk/v2/go/common/util/cmdutil"
-	"github.com/pulumi/pulumi/sdk/v2/go/common/util/contract"
 )
 
+// TO-DO: Remove as part of Pulumi v3.0.0
```

Comment:
Can we add an issue to track and add a link to the TODO comment here? As an aside, I also wonder if we should add an issue tag in GH (or something) so we can more easily track what things we'd like to address in the next major revision. (To make it easier on ourselves when we start planning for such a release.)

Also as an aside, this command is marked as "[PREVIEW]". Since it's a preview, would we necessarily have to wait until the next major version to delete it?

Reply:
I'm going to suggest we leave it in CLI for now so that anyone who has been testing it actually can still use it and get the help text

I will create a follow issue for it's removal - we should discuss the tag with @leezen